vault/diagnose/tls_verification_test.go

```diff
@@ -309,9 +309,9 @@ func TestTLSInvalidMaxVersion(t *testing.T) {
 	}
 	err := ListenerChecks(listeners)
 	if err == nil {
-		t.Error("TLS Config check on fake certificate should fail")
+		t.Fatalf("TLS Config check on fake certificate should fail")
 	}
 	if err.Error() != fmt.Errorf(maxVersionError, "0").Error() {
-		t.Errorf("Bad error message: %w", err)
```

Comment:
i think error here was fine since nothing after this depended on the results of that check, but fatal is good too!